### PR TITLE
KAFKA-13786:  Optimized documentation for control.plane.listener.name parameter

### PR DESCRIPTION
**Pull request by RivenSun2**
Optimized documentation for control.plane.listener.name parameter

### CODE REVIEW

**Comment by RivenSun2** [comment]
Hi @showuon
please help to review the PR .
Thanks.

File: core/src/main/scala/kafka/server/KafkaConfig.scala [hunk 1]
```diff
@@ -766,7 +766,9 @@ object KafkaConfig {
     "listener.security.protocol.map = INTERNAL:PLAINTEXT, EXTERNAL:SSL, CONTROLLER:SSL\n" +
     "control.plane.listener.name = CONTROLLER\n" +
     "then controller will use \"broker1.example.com:9094\" with security protocol \"SSL\" to connect to the broker.\n" +
-    "If not explicitly configured, the default value will be null and there will be no dedicated endpoints for controller connections."
+    "If not explicitly configured, the default value will be null and there will be no dedicated endpoints for controller connections." +
+    s" And the value of <code>$InterBrokerListenerNameProp</code> will be used for controller connections.\n" +
```
Comment (showuon) [inline core/src/main/scala/kafka/server/KafkaConfig.scala]: After checking the code and docs, I found the line to get `interBrokerListenerName`, is actually trying to get inter broker listener name, OR `security.inter.broker.protocol`:
```
val controllerToBrokerListenerName = config.controlPlaneListenerName.getOrElse(config.interBrokerListenerName)
```

I'm still thinking the explanation is complicated and redundant. Could we just add the last line only. That is:
```
"If not explicitly configured, the default value will be null and there will be no dedicated endpoints for controller connections." +
 s"If explicitly configured, the value cannot be the same as the value of <code>$InterBrokerListenerNameProp</code>."
```
WDYT?

**Review by showuon — COMMENTED** [review]
@RivenSun2 , thanks for the PR. Left a comment.

File: core/src/main/scala/kafka/server/KafkaConfig.scala (hunk 1 above)
Comment (RivenSun2) [reply core/src/main/scala/kafka/server/KafkaConfig.scala]: sure

**Review by showuon — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**